internal/tiltfile/tiltfile_state.go

```diff
@@ -93,10 +95,11 @@ const (
 	dcResourceN    = "dc_resource"
 
 	// k8s functions
-	k8sYamlN     = "k8s_yaml"
-	filterYamlN  = "filter_yaml"
-	k8sResourceN = "k8s_resource"
-	portForwardN = "port_forward"
+	k8sYamlN              = "k8s_yaml"
+	filterYamlN           = "filter_yaml"
+	k8sResourceN          = "k8s_resource"
+	portForwardN          = "port_forward"
+	k8sExtraImageLocation = "k8s_extra_image_location"
```

Comment:
not thrillllled about this function name... `k8s_extra_image_location_for_kind`? (That's also not great... feel free to leave it for now if you can't come up w/ anything better.)

Reply:
Yeah, naming this thing is causing me grief. I'll come up with a few options and bounce them off people on slack.

Reply:
ok, this seems not as terrible as most options:
`k8s_kind('Environment', image_json_path='{.spec.runtime.image}')`